reviews two/swagger.yaml

```diff
@@ -0,0 +1,879 @@
+    ### Draft Orders API 
+      
+      This API is about operations you can perform on the draft order object as defind below:
+    
+    Field | Definition | Data Type
+      -----|-----------|-------------------------
+      `id`| `id` of the draft order |`string`
+      `order_id`  | The id of the order associated to the draft order, once created. | `string`        
+      `name` | Name of the draft order, format #D<number>, where number, is an sequential identifier unique to the shop, starting at 1. For example #D133 | `integer`
+      `customer` | Customer object will be serialized with only the default address, however only the ID can be set in order to associate the customer to the draft order. Setting the value to null removes the customer from the draft order. | `object`
+      `shipping_address` | The mailing address to where the draft order will be shipped. | `string`
+      `billing_address` | The mailing address associated with the payment method. | `string`
+      `note` | The text of an optional note that a shop owner can attach to the draft order. | `string`
+      `email` | The email address used for sending notifications. |`string`
+      `currency` | The three letter code for the currency to be used for the payment. | `string`
+      `invoice_sent_at` | DateTime when the invoice was emailed to the customer by Shopify. | `dateTime`
+      `invoice_url` | The url to send to the customer so that they can complete the checkout.  When using `send_invoice`, this url is emailed to the customer. This field can be used so that an API client can use another method of communication to provide the url to the customer. | `string`
+      `line_item`[ ] | | array of `line_item` objects
+      `metafields`[ ] | | array of `metafield` objects
+      `shipping_line` | | `object`
+      `tags` | Tags are additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. Each individual tag is limited to 40 characters in length. | `string`
+      `tax_exempt` | Sets whether taxes are exempt for this draft order. If this value is `false`, Shopify will honor the `tax_exempt` value for each `line_item`. | `boolean`
+      `tax_lines` | Tax lines describing the sum of each type of tax line for line items and shipping line. | array of `tax_line` objects
+      `discount` | Order level discount. | `string`
+      `taxes_included` | Shop settings taxes are included in the price | `boolean`
+      `total_tax` | Total tax amount | `integer`
+      `completed_at` | Date at which an order was created and the draft order moved to “completed” status. | `DateTime`
+      `created_at` | By default, this auto-generated property is the date and time when the order was created in Shopify, in ISO 8601 format. If you are importing orders to the Shopify platform from another system, the writable `processed_at` property will override the `created_at` date. | `dateTime`
+      `updated_at` | | `dateTime`
```

Comment:
The date and time when the draft order was last modified. The API returns this value in ISO 8601 format.